CovidWatch iOS/ViewControllers/Splash.swift

```diff
@@ -14,20 +14,48 @@ class Splash: UIViewController {
     @IBOutlet weak var descriptionText: UILabel!
     @IBOutlet weak var startButton: UIButton!
     @IBOutlet weak var btnHeight: NSLayoutConstraint!
-    
+
+    static let animationDuration = 1.0
+    static let animationSlowdown = 1.5
+    static let delayBetweenAnimations = 0.2
+
     override func viewDidLoad() {
         super.viewDidLoad()
         startButton.update(height: btnHeight)
+        // prepare for animation
+        self.startButton.alpha = 0.0
+        self.descriptionText.alpha = 0.0
         // accessibility identifiers
         setupAccessibilityAndLocalization()
-        
-        if UserDefaults.standard.checkIfStartedOnboarding {
+
+        if UserDefaults.standard.checkIfStartedOnboarding && UserDefaults.standard.isFirstTimeUser {
```

Comment:
This was causing the bug that took you to bluetooth instead of the home if you had finished onboarding.